GliaWidgets/Sources/Coordinators/EngagementCoordinator/EngagementCoordinator.Environment.Mock.swift

```diff
@@ -37,7 +37,8 @@ extension EngagementCoordinator.Environment {
         orientationManager: .mock(),
         proximityManager: .mock,
         log: .mock,
-        snackBar: .mock
+        snackBar: .mock,
+        operatorRequestHandlerService: .init(environment: .init(uiApplication: .mock, log: .mock), viewFactory: .mock())
```

Comment:
Since you added `.mock()` for `OperatorRequestHandlerService`, you can use it here

Reply:
Good catch, must have missed it :)

Reply:
fixed [42a1750](https://github.com/salemove/ios-sdk-widgets/pull/885/commits/42a1750b482abe42ffdc5bc2f1aed6576b5ff381)